components/compositing/compositor.rs

```diff
@@ -408,7 +408,9 @@ impl IOCompositor {
             version_string,
         };
 
-        compositor.assert_gl_framebuffer_complete();
+        let gl = &compositor.webrender_gl;
+        println!("Running on {}", gl.get_string(gleam::gl::RENDERER));
+        println!("OpenGL Version {}", gl.get_string(gleam::gl::VERSION));
```

Comment:
I added helper print here when the compositor is initialized.
It should be eaiser to get our GL infor when opening tickets.

Also could we remove the assert here, because platforms like GTK's GLArea won't attach the buffers until rendering. It's tricky to create compositor in this way.

Reply:
I suppose these `println` should be `info!` to avoid printing this for all embedders. We can always request the reporter to run with `RUST_LOG=info` if we suspect the issue is with GL.

Reply:
> Also could we remove the assert here, because platforms like GTK's GLArea won't attach the buffers until rendering

I am not sure, but @mrobinson or @jdm might now more about this.

Reply:
I'll remove it and work in other PRs instead. The prints have changed to info logs